.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Push on Tag
+
+# This workflow is triggered on new tag pushed to the repository.
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish-builder:
+    name: Publish Builder
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu18.04]
+    env:
+      OS_NAME: ${{ matrix.os }}
+      REGISTRY_NAME: registry.zilliz.com/milvus/milvus-operator
```

Comment:
We need to publish to Dockerhub, so the variable REGISTRY_NAME needs to be changed